### PR TITLE
Updated document with example

### PR DESCRIPTION
**Pull request by KishorFlux7**
Updated the doc_source with an example - How to create resources with PersonalAccessToken from secretsmanager.

### CODE REVIEW

**Review by deepsvc — APPROVED** [review]
This looks good and is essential

File: doc_source/update-github-repo.md [hunk 1]
```diff
@@ -32,10 +32,9 @@ Properties:
   RepositoryDescription: String
   PersonalAccessToken: String
   IsPrivate: Boolean
-  RepositoryOwner: String
```
Comment (mksha) [inline doc_source/update-github-repo.md:35]: is it intentional we are removing `RepositoryOwner` ? @KishorFlux7

**Review by mksha — COMMENTED** [review]
I saw, we are removing RepositoryOwner from docs, was that unused var?

File: doc_source/update-github-repo.md (hunk 1 above)
Comment (deepsvc) [reply doc_source/update-github-repo.md:35]: Yes, GitHub does not use that anymore. Anyone who creates a repo gets a contributor access and that is all.

**Comment by deepsvc** [comment]
> I saw, we are removing RepositoryOwner from docs, was that unused var?

Yup, that is right. It is not used anymore.

**Review by sirhc — APPROVED** [review]

**Review by sampritavh — APPROVED** [review]
LGTM

**Pull request merged**